CAIPs/caip-10.md

```diff
@@ -24,50 +24,54 @@ The motivation for proposal stem from designing a chain-agnostic protocol for co
 
 ## Specification
 
-The account id specification will be prefixed with the CAIP-2 blockchain ID and delimited with an at sign (`@`) commonly used to define addresses.
+The account id specification will be prefixed with the CAIP-2 blockchain ID and delimited with a colon sign (`:`)
 
 ### Syntax
 
 The `account_id` is a case-sensitive string in the form
 
 ```
-account_id:        account_address + "@" + chain_id
-account_address:   [a-zA-Z0-9]{1,64}
+account_id:        chain_id + ":" + account_address
 chain_id:          [:-a-zA-Z0-9]{5,41}
+account_address:   [a-zA-Z0-9]{1,64}
 ```
 
 ### Semantics
 
-The `account_address` is a case sensitive string which its format is specific to the blockchain that is referred to by the `chain_id`
 The `chain_id` is specified by the [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) which describes the blockchain id.
+The `account_address` is a case sensitive string which its format is specific to the blockchain that is referred to by the `chain_id`
 
 ## Rationale
 
 The goals of the general account ID format is:
 
 - Uniqueness between chains regardless if they are mainnet or testnet
-- Readibility using `@` to easily identify the account address
+- Readibility using the prefix of a chainId to quickly identify before parsing the address
 - Restricted to constrained set of characters and length for parsing
 
+## Backwards Compatibility
+
+Previously CAIP-10 schema was defined by appending as suffix the CAIP-2 chainId delimited by the at sign (`@`)
```

Comment:
I've added this section to comment on the previous schema. cc @oed

Reply:
I think an example would be good here to remove any ambiguity.

Reply:
I've moved the "backwards compatibility" section below the test cases and added a legacy example